docs/user-guide/rug/conventions.md

```diff
@@ -101,288 +70,262 @@ the [`CHANGELOG.md` format][changelog].
 
 [changelog]: http://keepachangelog.com/
 
-### Rug DSL and TypeScript Files
+### Project Naming
 
-Rug files should have the `.rug` extension when written using the
-DSL and the standard TypeScript `.ts` extension when written in
-TypeScript.  You can safely intermix Rug DSL and TypeScript Rugs in
-the same project.  Rug files should placed in the following locations
-within a _Rug project_.  Rug editors and generators should be in
-`.atomist/editors` and Rug reviewers should be in `.atomist/reviewers`.
+A Rug project name should be hyphenated and start with the technology
+stack being targeted, such as `spring-boot`, followed by `-editors` if
+this is a Rug project with a collection of useful editors or reviewers.
 
-BDD-style tests for your Rugs are strongly recommended and should be
-located within a `.atomist/tests` directory and have the `.rt`
-extension.
+If the main purpose of a Rug project is to be a generator, then ending
+the name of the type of project it will generate would be most
+appropriate.  For example if your Rug projects's main purpose was to be
+a generator for a Spring Boot Rest Service then
+`spring-boot-rest-service` or even `java-spring-boot-rest-service`
+would be appropriate. If your Rug projects's main purpose was simply to
+generate a valid Maven project then `maven-project` would be appropriate.
 
-If your Rugs, typically editors, use any templates, they are placed in
-the `.atomist/templates` directory.
+### Configuration File
 
-### Rug Naming
+Rug project configuration is stored in a file in the `.atomist/manifest.yml`
+file. It describes the Rug project according to the following rules:
 
-Rug editors, generators, reviewers should have their names
-formatted using [UpperCamelCase][ucc].
+*   `#!yaml group`: The organisation behind this Rug project. Most commonly
+    the GitHub organisation in which they reside.
 
-[ucc]: http://wiki.c2.com/?UpperCamelCase
+*   `#!yaml artifact`: Name of the Rug archive (see next section)
 
-Following the rule of making implicit concepts explicit, the name of
-your Rug should correspond to a complete and specific description of
-the purpose of the Rug.
+*   `#!yaml version`: [Semantic version][semver] of this Rug project
 
-For example, `AddDocker` is a good name if the Rug adds Docker to
-anything, but `AddDockerToMavenProjects` is better if the intention of
-the Rug is to only work with projects that follow Maven conventions.
+[semver]: http://semver.org/
 
-While a Rug `.rug` file can contain many different editors,
-generators and reviewers, the Rug runtime
-enforces that the first Rug definition in a `.rug` file should match
-the name of the file itself. Therefore Rug files should be formatted
-using UpperCamelCase to match the name of the first Rug definition in
-the file.
+### Dependency Settings
 
-Further, Rugs within the same file are conventionally understood to be
-in support of the main and initial Rug in the file.
+#### Runtime dependency
```

Comment:
We probably do not need subsections here. If you feel strongly that they should be kept, I'd move have some text between the two headers explaining that there are two types of dependencies.